openmc/plotter.py

```diff
@@ -59,7 +59,7 @@ def _get_legend_label(this, type):
     """Gets a label for the element or nuclide or material and reaction plotted"""
     if isinstance(this, str):
         return f'{this} {type}'
-    elif this.name is '':
+    elif this.name == '':
```

Comment:
I see you were getting the same warning about comparison to a literal as I was 😃 

Thanks for taking care of it!